bindata/v4.1.0/kube-controller-manager/pod.yaml

```diff
@@ -21,7 +21,48 @@ spec:
         echo -n "."
         sleep 1
       done
+  - name: wait-for-cpc-host-port
+    terminationMessagePolicy: FallbackToLogsOnError
+    image: ${IMAGE}
+    imagePullPolicy: IfNotPresent
+    command: ['/usr/bin/timeout', '30', "/bin/bash", "-c"]
+    args:
+    - |
+      echo -n "Waiting for port :10357 to be released."
+      while [ -n "$(lsof -ni :10357)" ]; do
+        echo -n "."
+        sleep 1
+      done
   containers:
+  - name: cluster-policy-controller-REVISION
+    image: registry.svc.ci.openshift.org/ocp/4.3:cluster-policy-controller
```

Comment:
better/more correct way to reference `cluster-policy-controller` image? This unblocks my testing for now